README.md

```diff
@@ -10,268 +10,475 @@ hard-to-find defects early in the application development lifecycle.
 
 The CodeSonar plugin collects project analysis data from a designated CodeSonar hub.
 
-Historical data about CodeSonar warning counts and code size are presented in the Job Dashboard.
-
 The plugin can be configured to change the build result if the CodeSonar analysis results meet specified conditions.
 The following documentation cites relevant sections in the CodeSonar manual. These citations take the form:
 
-**MANUAL**: **Subject** > ... > **Page Title**
+**MANUAL:** *Subject > ... > Page Title*
+
+where *Subject > .... > Page Title* denotes a navigation path through the CodeSonar manual table of contents.
+
+### About CodeSonar ®
+
+[CodeSonar](https://www.grammatech.com/codesonar-cc), GrammaTech's flagship static analysis software, identifies programming bugs that can result in system crashes, memory corruption, leaks, data races, and security vulnerabilities.
 
-where **Subject** > **....** > **Page Title** denotes a navigation path through the CodeSonar manual table of contents.
+By analyzing both source code and binaries, CodeSonar empowers developers to eliminate the most costly and hard-to-find defects early in the application development lifecycle.
```

Comment:
This section duplicates the very first section of this document which starts with `## About CodeSonar` (level 2 header).  Seems like the duplication was unintentional, but I am not sure how they should be merged.